unit_tests.cpp

```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <assert.h>
+#include "quadro_solve.cpp"
+
+struct simple_test {
+    double a = 0;
+    double b = 0;
+    double c = 0;
+    double x1 = 0;
+    double x2 = 0;
+    int num_of_sol = 0;
+};
+
+void unit_tests(simple_test simpleTest[], int num_of_tests = 0) {
```

Comment:
Your naming convention is not consistent 🤔 . Before you named functions with camelCase and here you use snake_case. For NumOfSolutions you used PascalCase, but for simple_test you used snake_case. For variables you for some reason use camelCase and snake_case in the same line.